options/options.c

```diff
@@ -715,6 +715,10 @@ const m_option_t mp_opts[] = {
                       0, INT_MAX, ({"auto", -1})),
 #endif
 
+#if HAVE_VAAPI_HWACCEL
+    OPT_STRING("vaapi-device", vaapi_device, M_OPT_FILE),
```

Comment:
Maybe the option could be named `--decode-device` and combined with CUDA https://github.com/mpv-player/mpv/pull/4470 somehow? (just a suggestion, no idea if it can be really done)